src/main/java/com/jcabi/dynamo/mock/H2Data.java

```diff
@@ -319,4 +325,13 @@ private static String value(final AttributeValue attr) {
         return val;
     }
 
+    /**
+     * Base32-encodes table name for use with H2.
+     * @param table Table name to encode
+     * @return Base-32-encoded table name
+     */
+    private String encodeTableName(final String table) {
+        return new Base32(true, (byte) '_').encodeAsString(table.getBytes());
```

Comment:
output of Base32 contains padding symbol ```=``` (see http://tools.ietf.org/html/rfc4648#section-6)
which is not allowed by AWS. Please, change padding symbol to underscore ```_``` after Bas32 conversion.
Don't forget to test